bindsnet/network/nodes.py

```diff
@@ -188,7 +188,7 @@ def step(self, inpts, dt):
         self.v.masked_fill_(self.s, self.reset)
 
         # Integrate input and decay voltages.
-        self.v += inpts
+        self.v = self.v.masked_fill(self.refrac_count  ==0, self.v + inputs)
```

Comment:
`masked_fill` can't be used here because it expects its second argument to be a single `float`, not a `torch.Tensor`.